### DISCUSSION TITLE
Power Limit for Farming

### DISCUSSION THREAD

**Discussion by dolomitt** [madMAx43v3r/chia-gigahorse#294] [General]
Hello there. I'm trying to understand what power limit I can use for my Nvidia card. I have a 3060 TI with 8GB and farming in Linux with chia-gigahorse-farmer around 100TB of C30 fresh from the oven. Can I limit the power of my card to 130 watts (nvidia-smi pl parameter) instead of its initial 200 watts? Would that have an impact on the performance for farming?

**Comment by artur5236** [comment]
I set 100W on 3060 ti and works fine

**Comment by Jacek-ghub** [comment]
It depends on the number of plots / compression level, right?  However, 100TB c30 for 3060 it is basically zilch, so most likely you can go as low as the card permits.  Also, if you are using remote-compute-server (maybe it is always used, not sure), swap that binary for the latest one, as it should have some improvements for c30 processing (if not already on the latest GH).

Just watch your nvtop to not have compute spillage over multiple challenges.

**Comment by dolomitt** [comment]
I tried 100 watts as mentioned by previous poster and no issue that I can see. thank you for the advices.
